src/jvm/main/org/jetbrains/kotlinx/lincheck/CTestConfiguration.kt

```diff
@@ -38,8 +38,10 @@ abstract class CTestConfiguration(
     val customScenarios: List<ExecutionScenario>
 ) {
     abstract fun createStrategy(
-        testClass: Class<*>, scenario: ExecutionScenario, validationFunction: Actor?,
-        stateRepresentationMethod: Method?, verifier: Verifier
```

Comment:
The code for checking invocation result was duplicated in stress and managed strategies. 
Once I refactored to be the extension function of `Strategy` class, there is no longer a need to pass the verifier in the constructor, since it effectively becomes unused by the concrete `Strategy` implementers.